DOCS userhelp section for trusted devices (#385)
silverstripe/silverstripe-mfa: docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md

title: Remember trusted devices
summary: Steps to stay signed in on a trusted device

# Remember trusted devices

> In order to avoid repeat requests to log in with multi-factor authentication through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log-in screen.

Reviewer: ```suggestion
In order to avoid repeat requests to log in with Multi-factor authentication (MFA) through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log-in screen.
```

> In order to avoid repeat requests to log in with multi-factor authentication through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log-in screen.

Reviewer: ```suggestion
In order to avoid repeat requests to log in with MFA through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log in screen.
```

Reviewer: Since the main document explains what Multi-factor authentication already is we can just use the acronym.

Reviewer: In this case I think possibly 'log in' would be the correct terminology?

## Marking you current device as trusted

> 1. Visit the log-in screen.

Reviewer: ```suggestion
1. Visit the log in screen.
```

2. In addition to entering your Email and Password, choose the option to **Keep me signed in**.

> 3. Continue to log in with your registered multi-factor authentication method.

Reviewer: ```suggestion
3. Continue to log in with your registered MFA method.
```

![A screenshot of the log-in screen, highlighting the checkbox to keep the user signed in](../_images/01-06-1-keep_me_signed_in.png)

## How trusted devices work

When logging in with the **Keep me signed in** option selected, a time-based cookie will be saved to the web browser being used by your current device which will act to remember you in subsequent visits to your site.

In addition to remembering the device, your username and password will also be remembered.

<div class="notice" markdown='1'>
It is important that you only choose to use this option on a device you know to be secure. A secure device may be one that requires you to log in before being able to access the Internet, for example one within an office network, or a personal mobile device with a passcode.

> A device shared between multiple people would not be considered secure and you should not use this option.

Reviewer: ```suggestion
**A device shared between multiple people would not be considered secure and you should not use this option.**
```

</div>

> This functionality is provided by default with the SilverStripe CMS however it may be disabled by a Developer in some projects.

Reviewer: ```suggestion
This functionality is provided by default with Silverstripe CMS however it may be disabled by a Developer in some projects.
```
